test/functional/cmdLineTests/pltest/playlist.xml

```diff
@@ -144,6 +147,13 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 	</test>
 	<test>
 		<testCaseName>cmdLineTester_pltest_tty_extended</testCaseName>
+		<disabled>
+			<comment>github.com/eclipse/openj9/issues/1511</comment>
```

Comment:
should we have https? If we have the same format, it will be easy to do auto-remove later. The same for the rest url

Reply:
Thanks, fixed